test/copyimg/copyimg.go

```diff
@@ -110,7 +110,9 @@ func main() {
 				logrus.Errorf("error opening storage: %v", err)
 				os.Exit(1)
 			}
-			defer store.Shutdown(false)
+			defer func() {
+				_, _ = store.Shutdown(false)
```

Comment:
silly lint is complaning about this :confused: does it still complain if you just write `store.Shutdown(false)` here? maybe it complaints just after a `defer`